BurgerShack.cpp

```diff
@@ -0,0 +1,99 @@
+#include "BurgerShack.h"
+
+BurgerShack::BurgerShack() : 
+numBurgersServedPerDay(576), numToppingOptions(12), profitPerWeek(745.32f), beefUsedPerWeek(47.5f)
+{
```

Comment:
Call your other constructor from this constructor
I explained the Delegating Constructor concept in another comment
- [ ] revise